api/v1beta1/azurecluster_default_test.go

```diff
@@ -24,8 +24,73 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
+func TestControlPlaneEndpointDefault(t *testing.T) {
```

Comment:
what is this test testing? We're not setting any defaults for the control plane endpoint in the webhook AFAIK

Reply:
This was just a test to verify that the Host and Port could be set independently, if it doesn't make sense to keep that around I can delete that test.

Reply:
let's remove it